preview_mode.ipynb

```diff
@@ -574,53 +235,88 @@
-        "cols = 1 #param {type:\"slider\", min:1, max:7, step:1}\n",
-        "rows = 1 #param {type:\"slider\", min:1, max:4, step:1}\n",
-        "\n",
-        "#markdown Should we calculate one style for the entire style image? Or should we \n",
-        "#markdown divide the style image into tiles just like we do with the the content?\n",
-        "use_tiled_style_image = False #param {type:\"boolean\"}\n",
-        "\n",
-        "#markdown Edge size in final image. Basically this is the amount you cut off the \n",
-        "#markdown intersecting edges of each tile. Set to 0 for no edge correction. 4 or 8\n",
-        "#markdown work real good.\n",
-        "edge_size = 8 #param {type:\"slider\", min:0, max:32, step:1}\n",
-        "\n",
-        "#markdown New experimental mode of blending out the edges. Renders \"grout\" tiles\n",
-        "#markdown (you know, at the intersections of content tiles) and then mixes those\n",
-        "#markdown pixels with those of the sewn-together image. This will nearly double\n",
-        "#markdown the runtime but it's good.\n",
-        "use_fluid_blend = False #param {type:\"boolean\"}\n",
-        "\n",
-        "#markdown Sigma magnitude for fluid blend - lower # means smoother transition\n",
-        "magnitude = 2 #param {type:\"slider\", min:1, max:99}\n",
-        "\n",
-        "#markdown You're encouraged to play with the different values of `content_blending_ratio`.\n",
-        "content_blending_ratio = 0.5 #@param {type:\"slider\", min:0, max:1, step:0.1}\n",
-        "\n",
-        "with StepTimer('Render preview image'):\n",
-        "  result = render(\n",
-        "      google_drive_base,\n",
+        "#@markdown Content images are the top row, style images are the first column. \n",
+        "\n",
+        "\n",
+        "#\n",
+        "# Preview Grid Parameters\n",
+        "#\n",
+        "gdrive_basedir = '/content/drive/MyDrive/images/' #@param\n",
+        "\n",
+        "content_folder = 'content' #@param\n",
+        "style_folder = 'style' #@param\n",
+        "\n",
+        "content_blending_ratio = 0.2 #@param {type:\"slider\", min:0, max:1, step:0.1}\n",
+        "\n",
+        "preview_width = 15 #@param {type:\"slider\", min:5, max:15, step:1}\n",
+        "preview_height = 15 #@param {type:\"slider\", min:5, max:15, step:1}\n",
+        "\n",
+        "gdrive_outdir = '/content/drive/MyDrive/images/previews/' #@param\n",
+        "\n",
+        "#\n",
+        "# Render Preview Grid\n",
+        "#\n",
+        "\n",
+        "content_imgs = os.listdir(gdrive_basedir + content_folder)\n",
+        "style_imgs =  os.listdir(gdrive_basedir + style_folder)\n",
+        "\n",
+        "content_len = len(content_imgs)\n",
+        "style_len = len(style_imgs)\n",
+        "\n",
+        "# initialize preview grid\n",
+        "preview_grid, subplots = plt.subplots(content_len+1, style_len+1)\n",
```

Comment:
This broke for me when trying to run 2 content x 4 style. I was able to fix it by doing this:
```suggestion
        "preview_grid, subplots = plt.subplots(style_len+1, content_len+1)\n",
```